solutions/usaco-831.mdx

```diff
@@ -168,6 +279,8 @@ int main() {
 	written << winners[2].size() << endl;
 }
 ```
-
 </CPPSection>
+<JavaSection>
+Unfortunately, implementing this solution in java is quite tricky, and is beyond the scope of Bronze
```

Comment:
bruh.

tbh, it's better to add the solution and an info block saying that it uses concepts from silver.

Reply:
This probably isn't the right pr to even include a java solution tbh. I can create another one for that.